pkgs/development/libraries/rdkafka/default.nix

```diff
@@ -1,17 +1,17 @@
-{ lib, stdenv, fetchFromGitHub, zlib, zstd, pkg-config, python3, openssl }:
+{ lib, stdenv, fetchFromGitHub, zlib, zstd, pkg-config, python3, openssl, which }:
 
 stdenv.mkDerivation rec {
   pname = "rdkafka";
-  version = "1.8.2";
+  version = "1.9.1";
 
   src = fetchFromGitHub {
     owner = "edenhill";
     repo = "librdkafka";
     rev = "v${version}";
-    sha256 = "sha256-YagvXeusHThUo5/1mMs+r+Nr03vAagdnFMkwX3hJsq4=";
+    sha256 = "sha256-r5H02HLqiixbShgXDEaYEe4OrQK2En5zuLtMOajEIBM=";
   };
 
-  nativeBuildInputs = [ pkg-config python3 ];
+  nativeBuildInputs = [ pkg-config python3 which ];
 
   buildInputs = [ zlib zstd openssl ];
 
```

Comment:
I don't see what it would bring us. It's a new dependency in the configure phase, that's it. Once we start patching things, it creates more work to update the library.